### PR TITLE
Options screen

### PR DESCRIPTION
**Pull request by EmilyCoulson**
[Jira Ticket](https://csc2058.atlassian.net/browse/KAN-30)

Created the options screen that has the setttings for the for music, volume, screen size, colour blind mode and language.

### CODE REVIEW

File: MakingItBetterG55/src/application/OptionsMenu.java [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+import javafx.scene.text.Text;
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyEvent;
+import javafx.scene.layout.Pane;
+import javafx.scene.shape.Rectangle;
+
+public abstract class OptionsMenu extends Application{
```
Comment (Zachary-i) [inline MakingItBetterG55/src/application/OptionsMenu.java]: All looks really good Emily! Could you switch from extending Application and having a separate stage for the options screen to creating and returning a scene and then in Main/ anywhere the options button is clicked just set scene to your scene? This is how we're doing the rest of the screen changes and I just want to be consistent

File: MakingItBetterG55/src/application/OptionsMenu.java [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+        optionsScreen.add(title, 16, 4);
+        
+        //back button
+	Button back = new Button("Back");
+	back.setStyle("-fx-cursor: hand; -fx-border-color: #152546; -fx-border-width: 14px; -fx-background-color: #536F7B; -fx-text-fill: white;");
+	back.setFont(Font.font("SansSerif", FontWeight.BOLD, 40));
+	back.setPrefSize(320, 29);
+	optionsScreen.add(back, 28, 32, 32, 32);
+	back.setOnAction(event -> {
+		stage.close();
```
Comment (Zachary-i) [inline MakingItBetterG55/src/application/OptionsMenu.java]: If you have this function return a scene you can take as an argument whatever scene and the stage that called the settings scene so you could write: stageThatCalledTheOptionsScreen.setScene(sceneThatCalledTheOptionsScreen); (with different variable names of course) which would take you back to whatever scene in the game you were on before going to the options screen

**Review by Zachary-i — CHANGES_REQUESTED** [review]
Good job Emily only a couple requested changes

**Review by goskelly — COMMENTED** [review]
Great work Emily! Just fix the things Zachary mentioned in his comment :)

**Pull request merged**